.travis.yml

```diff
@@ -126,7 +126,7 @@ install:
       if [ "$DO_COVERAGE" = "codecov" ]; then
         test -f build/luacov.stats.out || { echo 'build/luacov.stats.out does not exist.'; return 1; }
         luacov || return 1
-        travis_retry bash /tmp/codecov-bash -f build/luacov.stats.out -X gcov -X coveragepy -c -F "$1,${LUANAME//./}" || return 1
+        travis_retry bash /tmp/codecov-bash -f build/luacov.stats.out -X gcov -X coveragepy -F "$1,${LUANAME//./}" || return 1
```

Comment:
Strange that the `rm` below did not fail before?!  Was it not discovered correctly after all?!
(failing build before this commit: https://travis-ci.org/awesomeWM/awesome/jobs/477361809).

Reply:
I've confused `luacov.report.out` and `luacov.stats.out`.